app/src/main/java/com/babylon/wallet/android/domain/model/assets/AccountWithAssets.kt

```diff
@@ -59,6 +59,17 @@ data class Assets(
         }
     }
 
+    val ownedResources: List<Resource> by lazy {
```

Comment:
Maybe rename to `knownResources`, since owned has the meaning of non-zero balance.

Reply:
totally agree!